### PR TITLE
fix(textAngularManager): Fixes #763

### LINKED ISSUE #763: Regression with using multiple instances of textAngular on the same page
The recent version of textAngular (since Rangy became a hard dependency) appeared to introduce a regression when using multiple instances of textAngular on the same page.

**Plnkr examples:**
* [textAngular v1.2.2](http://plnkr.co/edit/N5ZhBPNXE9WVfb58YG7S?p=preview)
* [textAngular v1.3.7](http://plnkr.co/edit/5SKbenboNQCEsTv91xSu?p=preview)

**Steps to reproduce:**
* Add 2 instances of textAngular to a single page.
* Set the focus on the first directive.
* Set the focus on the second directive.
* Set the focus back to the first directive.
* Write something.
* Click on any of the buttons in the toolbar.
* *Note how the focus is lost (and the changes are discarded) in the recent version of textAngular.*

### PR DESCRIPTION
**Pull request by MrJaeger**
Fixes #763.  There is another PR out that addresses this issue ( #940 ), but I believe my solution is slightly more elegant and has passing tests.

### CODE REVIEW

**Comment by pbassut** [comment]
Give me a minute to test it. Please

**Comment by MrJaeger** [comment]
hey @pbassut have you had a chance to review this yet?

File: src/main.js [hunk 1]
```diff
@@ -868,17 +868,16 @@ textAngular.service('textAngularManager', ['taToolExecuteAction', 'taTools', 'ta
 		},
 		// this is used by taBind to send a key command in response to a special key event
 		sendKeyCommand: function(scope, event){
-			angular.forEach(editors, function(_editor){
```
Comment (pbassut) [inline src/main.js]: Won't removing this run the code only for one editor only instead of all of them?

**Comment by pbassut** [comment]
I made some comments in your PR. Thanks for the PR btw.

File: src/main.js (hunk 1 above)
Comment (MrJaeger) [inline src/main.js]: I think that is the correct behavior.  If i have two textAngular divs on the same page and I hit `Tab` in one of them (to trigger `indent`), why would I want that command to be sent to both divs?

Comment (pbassut) [inline src/main.js]: Yeah. In the case of tab this is correct. I'm wondering if sendKeyCommand is used for some other purpose that should be triggered in all instances. A for like that doesn't go without a reason. Any thoughts? 

Anyway, I'll search for where it is used and see how it is used. Give me a few minutes.

**Pull request merged**